test/test_convolutional_layer.h

```diff
@@ -209,27 +209,6 @@ TEST(convolutional, with_stride) {
 // test for AVX backends
 
 #ifdef CNN_USE_AVX
-TEST(convolutional, fprop_avx) {
```

Comment:
We didn't need this separately since a long time. I checked other layer tests.. Similar to what they have, `convolution.forward` test should test correct on few AVX jobs of Travis.